Tests/SQLKitTests/SQLKitTests.swift

```diff
@@ -209,47 +209,52 @@ final class SQLKitTests: XCTestCase {
 
     func testAltering() throws {
         // SINGLE
+        try db.alter(table: "alterable")
+            .rename(to: "new_alterable")
+            .run().wait()
+        XCTAssertEqual(db.results[0], "ALTER TABLE `alterable` RENAME TO `new_alterable`")
+
```

Comment:
Could you move this to the bottom of the function, would revert changing the existing asserts so it's clear they don't change them

Reply:
Definitely. Done.